generate_outputs.py

```diff
@@ -249,15 +257,17 @@ def demog_labels(code_scheme, x, y):
                 "rqa_s01e06_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E06),
                 "rqa_s01e07_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E07),
                 "rqa_s01e08_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E08),
+                "rqa_s01e09_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E09),
+                "rqa_s01e10_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E10),
 
                 "rqa_s01e01_raw": FoldStrategies.concatenate,
                 "rqa_s01e02_raw": FoldStrategies.concatenate,
                 "rqa_s01e03_raw": FoldStrategies.concatenate,
                 "rqa_s01e04_raw": FoldStrategies.concatenate,
                 "rqa_s01e05_raw": FoldStrategies.concatenate,
                 "rqa_s01e06_raw": FoldStrategies.concatenate,
-                "rqa_s01e07_raw": FoldStrategies.concatenate,
```

Comment:
why not 7 and 8?

Reply:
This was erroneously deleted thanks for catching it. Added back